great_expectations/core/batch_spec.py

```diff
@@ -32,6 +32,16 @@ def ge_load_time(self):
         return self.get("ge_load_time")
 
 
+class PandasBatchSpec(BatchSpec, metaclass=ABCMeta):
+    @property
+    def reader_method(self) -> str:
+        return self.get("reader_method")  # type: ignore[return-value]
```

Comment:
Either this return should indicate that it could be none, or this should be updated so that it's never `None`.
```suggestion
    def reader_method(self) -> str:
        return self["reader_method"]
```